SPECS/tensorflow/tensorflow.spec

```diff
@@ -14,8 +14,10 @@ Patch1:         CVE-2024-6232.patch
 Patch2:         CVE-2024-8088.patch
 Patch3:         CVE-2024-3651.patch
 Patch4:         CVE-2024-35195.patch
-Patch5:		CVE-2024-5569.patch
-Patch6:		CVE-2024-6923.patch
+Patch5:		      CVE-2024-5569.patch
+Patch6:		      CVE-2024-6923.patch
```

Comment:
minor issue - can you remove the tab characters here?

Reply:
I got it.